script/SubmitSafeTxs.s.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.26;
+pragma abicoder v2;
+
+/* solhint-disable no-console*/
+import {console} from "forge-std/console.sol";
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Safe} from "@safe/Safe.sol";
+import {Enum} from "@safe/common/Enum.sol";
+
+using stdJson for string;
+
+/**
+ * @title SubmitSafeTxs
```

Comment:
The title should be a short description of the contract (but not just the contract name itself):
https://docs.soliditylang.org/en/latest/natspec-format.html#tags